config/config-feature-flags.yaml

```diff
@@ -78,7 +78,7 @@ data:
   # Setting this flag will determine which gated features are enabled.
```

Comment:
Should we switch the default value to `true` in v0.43.0 release or v0.44.0 release?

Reply:
cc @jerop

Reply:
v0.43, we have an issue tracking this - https://github.com/tektoncd/pipeline/issues/5155